modules/machine-config-node-disruption-config.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machine-config-node-disruption-config_{context}"]
+= Configuring node restart behaviors after machine config changes
+
+You can create a node disruption policy to define the machine configuration changes that cause a disruption to your cluster, and which changes do not.
+
+You can control how your nodes respond to changes in the configuration files in the `/var` or `/etc` directory, systemd units, or SSH keys.
+
+include::snippets/machine-config-node-disruption-actions.adoc[]
+
+.Prerequisite
+
+
+* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see _Enabling features using feature gates_.
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster prevents minor version updates. The `TechPreviewNoUpgrade` feature set cannot be disabled. Do not enable this feature set on production clusters.
+====
+
+.Procedure
+
+. Edit the `MachineConfiguration` object to define the node disruption policy:
++
+[source,terminal]
+----
+$ oc edit MachineConfiguration cluster -n openshift-machine-config-operator
+----
+
+. Add a node disruption policy similar to the following:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: MachineConfiguration
+metadata:
+  name: cluster
+  namespace: openshift-machine-config-operator
+  labels:
+    machineconfiguration.openshift.io/role: worker # <1>
```

Comment:
This configuration is not role specific, the policy is applied cluster wide, to all nodes. So this label would not do anything.